documentation/gnmi/design/DASH.json

```diff
@@ -0,0 +1,724 @@
+//
+
+"routing-types-config": [
+    {
+        "direct": { 
+            "routing-type": "direct",
+            "actions": [ // up to 4 actions
+                {
+                    "action-type": "direct"
+                }
+            ]
+        }
+    },
+    {
+        "vpc": {
+            "routing-type": "vpc",
+            "actions": [ // ONLY 1 action here! More then 1 action is not allowed if action-type used is "maprouting"
+                {
+                    "action-type": "maprouting" // NOTE: "maprouting" will be the ONLY action. It cannot be used with other actions
+                }
+            ]
+        }
+    },
+    {
+        "vpc-direct": {
+            "routing-type": "vpc-direct",
+            "actions": [ // up to 3 actions
+                {
+                    "action-type": "staticencap",
+                    "encap-type": "vxlan"
+                }
+            ]
+        }
+    },
+    {
+        "drop": {
+            "routing-type": "drop",
+            "actions": [ // up to 3 actions
+                {
+                    "action-type": "drop"
+                }
+            ]
+        }
+    },
+    {
+        "appliance": {
+            "routing-type": "appliance",
+            "actions": [ // up to 3 actions
+                {
+                    "action-type": "appliance"
+                }
+            ]
+        }
+    },
+    {
+        "privatelink": {
+            "routing-type": "privatelink",
+            "actions": [ // up to 3 actions
+                {
+                    "action-type": "4to6"
+                },
+                {
+                    "action-type": "staticencap",
+                    "encap-type": "nvgre",
+                    "vni-key": 100
+                }
+            ]
+        }
+    },
+    {
+        "privatelinknsg": {
+            "routing-type": "privatelinknsg",
+            "actions": [ // up to 3 actions
+                {
+                    "action-type": "4to6"
+                },
+                {
+                    "action-type": "staticencap",
+                    "encap-type": "nvgre",
+                    "vni-key": 100
+                },
+                {
+                    "action-type": "appliance",
+                }
+            ]
+        }
+    },
+    {
+        "servicetunnel": {
+            "routing-type": "servicetunnel",
+            "actions": [ // up to 3 actions
+                {
+                    "action-type": "4to6"
+                },
+                {
+                    "action-type": "staticencap",
+                    "encap-type": "nvgre",
+                    "vni-key": 100
+                }
+            ]
+        }
+    }
+],
+
+//
+// active routing types for mappings (active routing types modules)
+// list of active modules impacts memory area reserved for mappings and needs to be configured as part of initial card config
```

Comment:
What other per DPU initial configurations should be expected (e.g. global connection aging time, etc.)?